sources/packages/web/src/views/institution/LocationApplications.vue

```diff
@@ -1,20 +1,83 @@
 <template>
   <v-container>
-    <h2>{{ locationName }}</h2>
-    <v-sheet elevation="1" class="mx-auto">
+    <p class="text-muted font-weight-bold h3">{{ locationName }}</p>
+    <p class="font-weight-bold h2">Student Applications</p>
+    <v-sheet elevation="1" class="mx-auto mt-2">
       <v-container>
-        <v-row>
-          <v-col cols="8">
-            <h2 class="color-blue">Applications</h2>
-          </v-col>
-        </v-row>
+        <p class="color-blue h3 font-weight-bold">New Applications</p>
+        <p>
+          New students have submitted applications for student aid funding or
+          grants. Confirm students are enrolled for the programs specificed in
+          their applications.
+        </p>
+
+        <DataTable
+          :autoLayout="true"
+          :value="applications"
+          class="p-m-4"
+          :paginator="true"
+          :rows="10"
+        >
+          <Column field="firstName" header="Name">
+            <template #body="slotProps">
+              <span
+                >{{ slotProps.data.firstName }}
+                {{ slotProps.data.lastName }}</span
+              >
+            </template>
+          </Column>
+          <Column field="studyStartPeriod" header="Study Period">
+            <template #body="slotProps">
+              <span>
+                {{ dateString(slotProps.data.studyStartPeriod) }} -
+                {{ dateString(slotProps.data.studyEndPeriod) }}
+              </span>
+            </template></Column
+          >
+          <Column field="applicationNumber" header="Application #"></Column>
+          <Column field="pirStatus" header="Status">
+            <template #body="slotProps">
+              <Chip
+                v-if="slotProps.data.pirStatus === 'completed'"
```

Comment:
Please consider displaying the PIR status directly. If it needs to be upper case, please add a CSS style.